### ISSUE TITLE
The terraform-provider-opentelekomcloud_v1.36.11 plugin crashed

### ISSUE THREAD

**Issue by krkardos** [opentelekomcloud/terraform-provider-opentelekomcloud#2552]
Not able to run our terraform code with the latest provider, because there is some bug in it.

# Terraform provider version
- Installing opentelekomcloud/opentelekomcloud v1.36.11...
- Installed opentelekomcloud/opentelekomcloud v1.36.11 (self-signed, key ID *************)

# Affected Resource(s)
- opentelekomcloud_cce_cluster_v3
- opentelekomcloud_dcs_instance_v1
- opentelekomcloud_lb_loadbalancer_v3
- opentelekomcloud_dns_recordset_v2
- opentelekomcloud_ecs_instance_v1
- opentelekomcloud_nat_gateway_v2
- opentelekomcloud_rds_instance_v3
- opentelekomcloud_vpnaas_service_v2
- opentelekomcloud_vpnaas_ike_policy_v2
- opentelekomcloud_vpnaas_endpoint_group_v2

# Terraform Configuration Files
```hcl
terraform {
  required_providers {
    opentelekomcloud = {
      source = "opentelekomcloud/opentelekomcloud"
    }
  }
}
```
```

# Debug Output/Panic Output

Error: Plugin did not respond
│ 
│   with module.vpn.opentelekomcloud_vpnaas_ike_policy_v2.ike_policy["krzn1"],
│   on .terraform/modules/vpn/main.tf line 22, in resource "opentelekomcloud_vpnaas_ike_policy_v2" "ike_policy":
│   22: resource "opentelekomcloud_vpnaas_ike_policy_v2" "ike_policy" {
│ 
│ The plugin encountered an error, and failed to respond to the
│ plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may
│ contain more details.
╵
╷
│ Error: Request cancelled
│ 
│   with module.vpn.opentelekomcloud_vpnaas_ipsec_policy_v2.ipsec_policy["krzn1"],
│   on .terraform/modules/vpn/main.tf line 36, in resource "opentelekomcloud_vpnaas_ipsec_policy_v2" "ipsec_policy":
│   36: resource "opentelekomcloud_vpnaas_ipsec_policy_v2" "ipsec_policy" {
│ 
│ The plugin.(*GRPCProvider).UpgradeResourceState request was cancelled.

Error: Plugin did not respond
│ 
│   with module.vpn.opentelekomcloud_vpnaas_endpoint_group_v2.peer_group["krzn1"],
│   on .terraform/modules/vpn/main.tf line 59, in resource "opentelekomcloud_vpnaas_endpoint_group_v2" "peer_group":
│   59: resource "opentelekomcloud_vpnaas_endpoint_group_v2" "peer_group" {
│ 
│ The plugin encountered an error, and failed to respond to the
│ plugin.(*GRPCProvider).ReadResource call. The plugin logs may contain more
│ details.
╵
Stack trace from the terraform-provider-opentelekomcloud_v1.36.11 plugin:
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xfbab87]
goroutine 1262 [running]:
github.com/opentelekomcloud/terraform-provider-opentelekomcloud/opentelekomcloud/services/cce.resourceCCEClusterV3Read({0x193aa48?, 0xc000c1ef60?}, 0xc000c6cc80, {0x167eaa0?, 0xc0009ca8c0})
	github.com/opentelekomcloud/terraform-provider-opentelekomcloud/opentelekomcloud/services/cce/resource_opentelekomcloud_cce_cluster_v3.go:535 +0xba7
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).read(0xc000483880, {0x193aa80, 0xc000e1af90}, 0xd?, {0x167eaa0, 0xc0009ca8c0})
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.24.1/helper/schema/resource.go:724 +0x12e
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).RefreshWithoutUpgrade(0xc000483880, {0x193aa80, 0xc000e1af90}, 0xc000c6b5f0, {0x167eaa0, 0xc0009ca8c0})
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.24.1/helper/schema/resource.go:1015 +0x585
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ReadResource(0xc000132db0, {0x193aa80?, 0xc000e1ae70?}, 0xc00015ccc0)
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.24.1/helper/schema/grpc_provider.go:[613](https://gitlab.ms.otc.t-systems.com/logineo/logineo-otc-deployment/-/jobs/54004#L613) +0x4a5
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ReadResource(0xc00042cdc0, {0x193aa80?, 0xc000e1a810?}, 0xc0007fcf00)
	github.com/hashicorp/terraform-plugin-go@v0.14.1/tfprotov5/tf5server/server.go:748 +0x4b1
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ReadResource_Handler({0x1[619](https://gitlab.ms.otc.t-systems.com/logineo/logineo-otc-deployment/-/jobs/54004#L619)240?, 0xc00042cdc0}, {0x193aa80, 0xc000e1a810}, 0xc0000d3420, 0x0)
	github.com/hashicorp/terraform-plugin-go@v0.14.1/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:349 +0x170
google.golang.org/grpc.(*Server).processUnaryRPC(0xc00014c000, {0x1940420, 0xc000182820}, 0xc0009465a0, 0xc0004ef4a0, 0x22a99d0, 0x0)
	google.golang.org/grpc@v1.56.3/server.go:1335 +0xdf3
google.golang.org/grpc.(*Server).handleStream(0xc00014c000, {0x1940420, 0xc000182820}, 0xc0009465a0, 0x0)
	google.golang.org/grpc@v1.56.3/server.go:1712 +0xa36
google.golang.org/grpc.(*Server).serveStreams.func1.1()
	google.golang.org/grpc@v1.56.3/server.go:947 +0xca
created by google.golang.org/grpc.(*Server).serveStreams.func1
	google.golang.org/grpc@v1.56.3/server.go:958 +0x15c
Error: The terraform-provider-opentelekomcloud_v1.36.11 plugin crashed!
This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.

# Steps to Reproduce
Use the latest provider which was deployed today ( 1.36.11 ).

# Expected Behavior
Working terraform plan + apply.

# Actual Behavior
Provider crashed, we got plugin error messages.

# Important Factoids
Older provider versions are working fine, bug is only exists in the latest provider version. ( 1.36.11 )

**Comment by artem-lifshits** [comment]
@krkardos please check latest release, there was a bug within cluster v3 resource which is fixed in provider v1.36.12.

**Comment by krkardos** [comment]
> @krkardos please check latest release, there was a bug within cluster v3 resource which is fixed in provider v1.36.12.

Thanks Artem, now with the newest provider, everything is working! :)
